.travis.yml

```diff
@@ -108,6 +129,21 @@ matrix:
         - ${PIP} install cibuildwheel==0.10.2
       script:
         - cibuildwheel --output-dir dist
+
+    - name: osx-py38
+      os: osx
+      language: generic
+      env:
+        - TOXENV=py38
+        - PIP=pip2
```

Comment:
Why is pip2 used on MacOS?

Reply:
I have no idea, it's a copy paster from the previous env

Reply:
OK, must have missed it in an earlier review. Thanks for updating!